### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [Rem0o/FanControl.Releases#1416] [Ideas]
Hello there,
I assume it's not compatible with Linux yet, but I'm pretty sure that many Linux users would be happy to have access to this FanControl.

I would also be very thankful if I could run this FanControl on my Server :)

**Comment by Rem0o** [comment]
Pretty much none of the UI and core driver/control logic is Linux portable.  Would require pretty much a whole new project.

**Reply by 4e576rt8uh9ij9okp** [reply]
Ok understandable. Maybe one day a hero like you will arive and maybe find a way :p

**Comment by Koli0842** [comment]
I absolutely adore fancontrol, but the same cannot be said about windows. I don't think I'll ever switch to 11, which has me trying Linux currently. Coolercontrol looks like a nice project, and I got it detecting my fans, but I don't really like the UI at the moment. I wish it was possible to plumb the GUI of fancontrol to the backend of coolercontrol, or lm_sensors directly

**Reply by Rem0o** [reply]
It's a WPF app, so there is no simple port to linux possible.
